decidim-admin/app/controllers/decidim/admin/organization_homepage_content_blocks_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    # Controller that allows managing the organization homepage content blocks
+    class OrganizationHomepageContentBlocksController < Decidim::Admin::ApplicationController
+      layout "decidim/admin/settings"
+
+      helper_method :content_block
+
+      def edit
+        enforce_permission_to :update, :organization, organization: current_organization
+        @form = form(ContentBlockForm).from_model(content_block)
+      end
+
+      def update
+        enforce_permission_to :update, :organization, organization: current_organization
+        @form = form(ContentBlockForm).from_params(params)
+
+        UpdateContentBlock.call(@form, content_block, :homepage) do
+          on(:ok) do
+            redirect_to edit_organization_homepage_path
+          end
+          on(:invalid) do
+            render :edit
+          end
+        end
+      end
+
+      private
+
+      def content_block
+        @content_block ||= content_blocks.find_by(manifest_name: params[:id]) ||
+                           unused_manifests.find { |manifest| manifest.name.to_s == params[:id] }
+      end
+
+      def content_blocks
+        @content_blocks ||= Decidim::ContentBlock.for_scope(:homepage, organization: current_organization)
+      end
+
+      def used_manifests
```

Comment:
Maybe rename `used_manifests` to `current_manifests` and `unused_manifests` to `available_manifests`?

Although it's a bit weird to talk about manifests when we seem to want content blocks